packages/server/src/models/flower.entity.ts

```diff
@@ -0,0 +1,23 @@
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+import { UserShelfActions } from '../constants/user-shelf';
+
+@Entity()
+export class Flower {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  shelfId: number;
+
+  @Column()
+  name: string;
+
+  @Column()
+  description: string;
+
+  @Column('simple-array')
+  pictureUrls: string[];
+
+  @Column('simple-json')
+  rrules: { [key in UserShelfActions]: string }
```

Comment:
maybe rules?

Reply:
I call these `rrules` because these would be recurrence rules according to iCalendar standard https://icalendar.org/iCalendar-RFC-5545/3-3-10-recurrence-rule.html, genearated and parsed by https://github.com/jakubroztocil/rrule, probably with the help of https://github.com/Fafruch/react-rrule-generator or similar, so `rrule` is a pretty standard name for it.

Reply:
Looks good, thanks.